### PR TITLE
Allow admins to create and manage recommended tags

### PR DESCRIPTION
**Pull request by oestrich**
Includes an API to view the active ones

https://zube.io/smartlogic/bchd/c/227
https://zube.io/smartlogic/bchd/c/228
https://zube.io/smartlogic/bchd/c/229

### Migrations to Run: Yes / No

Yes

Screenshot from 2021-11-09 21-03-56

### CODE REVIEW

**Comment by danivovich** [comment]
Spec failure looks like your psql instance made the sql file something CI can't handle. Looks like CI is 10 and prod is 11, you are probably on 12 or 13

File: app/views/admin/recommended_tags/show.html.haml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+%h3
```
Comment (danivovich) [inline app/views/admin/recommended_tags/show.html.haml:1]: will it be clear here that this is a recommended tag and not a tag itself?

File: app/views/admin/recommended_tags/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+%h1 Tag Manager
```
Comment (danivovich) [inline app/views/admin/recommended_tags/index.html.haml:1]: Should this be Recommended Tag Manager?

File: app/views/admin/recommended_tags/_recommended_tag.html.haml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+          = f.check_box :active, onchange: "javascript:submit()"
+          %span.slider.round
+    - else
+      = recommended_tag.active ? "Active" : "Inactive"
+
+  %td
+    %span
+      = link_to fa_icon("edit"), edit_admin_recommended_tag_path(recommended_tag.id), class: "tag_edit_icon"
+    %span
+      = link_to fa_icon("trash"), admin_recommended_tag_path(recommended_tag.id), method: :delete, data: { confirm: "Are you sure to delete #{recommended_tag.name} tag?" }, class: "tag_delete_icon"
```
Comment (danivovich) [inline app/views/admin/recommended_tags/_recommended_tag.html.haml]: should this message confirm they are deleting a recommended tag, not a tag?

File: app/views/admin/tags/index.html.haml [hunk 4]
```diff
@@ -1,4 +1,8 @@
-%h1 Tag Management
+%h1 Tag Manager
```
Comment (danivovich) [inline app/views/admin/tags/index.html.haml:1]: :+1:

File: db/structure.sql [hunk 5]
```diff
@@ -65,23 +51,23 @@ CREATE FUNCTION public.fill_search_vector_for_location() RETURNS trigger
 
 SET default_tablespace = '';
 
-SET default_with_oids = false;
+SET default_table_access_method = heap;
```
Comment (danivovich) [inline db/structure.sql]: This appears to be the CI problem

**Review by danivovich — CHANGES_REQUESTED** [review]
Looks great, I have some questions about clarity and the CI issue

File: app/views/admin/recommended_tags/index.html.haml (hunk 2 above)
Comment (oestrich) [reply app/views/admin/recommended_tags/index.html.haml:1]: The designs have "Tag Manager" for both tabs so I left it it at that

https://zube.io/files/smartlogic/1c3a710262dbb20f1cd5a643474f9f37-image.png

File: app/views/admin/recommended_tags/show.html.haml (hunk 1 above)
Comment (oestrich) [reply app/views/admin/recommended_tags/show.html.haml:1]: Maybe, but I updated it regardless

Screenshot from 2021-11-10 18-04-25

**Comment by oestrich** [comment]
Got postgres 10 running locally in docker and re-did the migrations. Here we go CI :crossed_fingers:

**Review by danivovich — APPROVED** [review]

**Pull request merged**